### ISSUE TITLE
Missing helper _interopRequireWildcard

### ISSUE THREAD

**Issue by rvolgers** [swc-project/swc#225]
Tested with swc 1.0.14

```
user@dev:~/$ cat foo.js
'use strict';
import * as bar from './bar';
user@dev:~/$ cat bar.js
export const a = 1;
export const b = 2;
user@dev:~/$ swc foo.js
'use strict';
var bar = _interopRequireWildcard(require('./bar'));
'use strict';
```

This is based on actual code that expects to be able to access `bar.a` and `bar.b` afterward in `foo.js`.

**Comment by kdy1** [comment]
What's your `.swcrc`?

**Comment by rvolgers** [comment]
```
{
  "jsc": {
    "parser": {
      "syntax": "ecmascript",
      "jsx": true,
      "classProperty": true,
      "functionBind": true,
      "decorators": true,
      "decoratorsBeforeExport": false
    }
  },
  "module": {
    "type": "commonjs",
    "strict": false,
    "strictMode": true,
    "lazy": false,
    "noInterop": false
  }
}
```

**Comment by kdy1** [comment]
Seems like it's already fixed in `1.0.15`. I'll publish `1.0.15` soon.

**Comment by kdy1** [comment]
@rvolgers I published 1.0.15

**Comment by rvolgers** [comment]
Unfortunately it doesn't appear to be fixed in 1.0.15.

```
user@dev:~/somepkg$ npm view @swc/core version
1.0.15
user@dev:~/somepkg$ cat foo.js
'use strict';
import * as bar from './bar';
user@dev:~/somepkg$ cat bar.js
export const a = 1;
export const b = 2;
user@dev:~/somepkg$ npx swc foo.js
var bar = _interopRequireWildcard(require('./bar'));
'use strict';

user@dev:~/somepkg$ cat .swcrc
 cat .swcrc
{
  "jsc": {
    "parser": {
      "syntax": "ecmascript",
      "jsx": true,
      "classProperty": true,
      "functionBind": true,
      "decorators": true,
      "decoratorsBeforeExport": false
    }
  },
  "module": {
    "type": "commonjs",
    "strict": false,
    "strictMode": true,
    "lazy": false,
    "noInterop": false
  }
}
```

**Comment by kdy1** [comment]
What is your nodejs version and os?
I'm asking this because there's [a test about it](https://github.com/swc-project/node-swc/blob/e4eb7f13ede136b91c6dc52bda7a2e486e5e92d1/__tests__/api_test.js#L5-L14) and [it passes](https://travis-ci.com/swc-project/node-swc/jobs/178371707) on 8 / 10 / 11

**Comment by kdy1** [comment]
I verified that it happens on osx 11 (although test successes)

**Comment by kdy1** [comment]
Fixed with`@swc/core@1.0.16`.
